### PR TITLE
i18n(pt-BR): Add `deploy.md` translation

### PR DESCRIPTION
**Pull request by Yan-Thomas**
#### What kind of changes does this PR include?

- Translated content

#### Description

Add PT-BR `deploy.md` page translation

### CODE REVIEW

File: src/pages/pt-br/guides/deploy.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+setup: import DeployGuidesNav from '~/components/DeployGuidesNav.astro';
+i18nReady: true
+---
+**Pronto para fazer a build e deploy do seu site Astro?** Siga um dos nossos guias para diferentes serviços de deploy ou role para abaixo para um guia geral sobre como fazer deploy de um site Astro.
+
+<DeployGuidesNav />
+
+## Opções de Deploy Rápido
+
+Você pode fazer build e deploy de um site Astro para diversos hosts rapidamente utilizando seja pela UI do painel dessa plataforma ou por uma interface de linha de comando.
```
Comment (caioferrarezi) [inline src/pages/pt-br/guides/deploy.md:14]:
```suggestion
Você pode fazer build e deploy de um site Astro para diversos hosts rapidamente utilizando ou a UI do painel dessa plataforma ou por uma interface de linha de comando.
```

File: src/pages/pt-br/guides/deploy.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+### UI do Website
+
+Uma forma rápida de fazer deploy do seu website é conectando o repositório Git online (e.x. GitHub, GitLab, Bitbucket) de seu projeto Astro a uma provedor de hospedagem e se aproveitando de continuous deployment utilizando o Git.
+
+Estas plataformas de host automaticamente detectam pushes no repositório fonte do seu projeto Astro, fazem a build do seu site e então fazem o deploy dele para a web em uma URL customizada ou em seu domínio pessoal. Geralmente, para configurar o deployment nessas plataformas você vai acabar seguindo passos como os seguintes: 
+
+1. Adicione seu repositório para um provedor Git online (e.x. GitHub, GitLab, Bitbucket)
+
+1. Escolha uma host que suporta **continuous deployment** (e.x. [Netlify]/pt-br/guides/deploy/netlify/) ou [Vercel](/pt-br/guides/deploy/vercel/)) e importe seu repositório Git como um novo site/projeto.
```
Comment (caioferrarezi) [inline src/pages/pt-br/guides/deploy.md:24]: all ordered lists are repeating the number one, does this get transformed on the final page?

Comment (Yan-Thomas) [reply src/pages/pt-br/guides/deploy.md:24]: Yes! They're rendered properly. Also, thanks for highlighting this line just found a broken link, going to fix this real quick 😅

Comment (caioferrarezi) [reply src/pages/pt-br/guides/deploy.md]: :sweat_smile: no problems!

**Review by caioferrarezi — APPROVED** [review]
LGTM

**Comment by Yan-Thomas** [comment]
Thanks for the review @caioferrarezi, amazing work! 💜

**Pull request merged**